students/15th/sunghoonkim/style/main.css

```
@@ -0,0 +1,317 @@
+}
+
+.nav-input-container input {
+  width: 100%;
+  height: 30px;
+  padding: 0px 25px;
+  border-radius: 3px;
+  border: 1px solid #dbdbdb;
+  outline: none;
+  background-color: #fafafa;
+}
+
+.nav-input-container input::placeholder {
+  opacity: 0.5;
+  text-align: center;
+}
+
+.nav-menuIcons {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.nav-menuIcons li {
+  margin: 0 10px;
+  font-size: 20px;
+  font-weight: 200;
+}
+
+.nav-menuIcons li i {
+  color: black;
+  opacity: 0.7;
+}
+
+main {
+  display: flex;
+  justify-content: center;
+  padding-top: 80px;
+  border-top: 1px solid #e6e6e6;
+}
+
+.feeds {
+  width: 650px;
+  margin-right: 30px;
+  background-color: white;
+  border: 1px solid #e6e6e6;
+  border-radius: 5px;
+}
+
+.feed-header {
+  display: flex;
+  justify-content: space-between;
+  padding: 10px 10px;
+}
+
+.feed-header-column {
+  display: flex;
+  align-items: center;
+}
+
+.feed-header img {
+  margin-right: 10px;
+}
+
+.fa-ellipsis-h {
+  font-size: 15px;
+}
+
+.feed-buttons {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 6px;
+}
+
+.feed-buttons i {
+  margin: 7px;
+  font-size: 25px;
+  opacity: 0.7;
+  cursor: pointer;
+}
+
+.feed-buttons .fa-heart {
+  color: red;
+  opacity: 1;
+}
+
+.feed-likes {
+  display: flex;
+  align-items: center;
+  padding: 0 13px;
+}
+
+.feed-likes img {
+  width: 25px;
+  margin-right: 10px;
+  border-radius: 50%;
+}
+
+.feed-contents {
+  margin-top: 17px;
+  padding: 0 13px;
+}
+
+.feed-comments {
+  margin-top: 10px;
+}
+
+.feed-comment {
+  padding: 0 13px;
+}
+
+.feed-comment-row {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 10px;
+}
+
+.feed-comment-row i {
+  opacity: 0.5;
+  cursor: pointer;
+}
+
+.feed-comment-timetag {
+  margin: 10px 0;
+  opacity: 0.5;
+}
+
+.feed-comment-input {
+  width: 100%;
+  display: flex;
+  align-items: center;
+  position: relative;
+}
+
+.feed-comment-input input {
+  width: 100%;
+  height: 60px;
+  padding-left: 13px;
+  padding-right: 45px;
+  border-style: none;
+  border-top: 1px solid #dbdbdb;
+  border-bottom-right-radius: 5px;
+  border-bottom-left-radius: 5px;
+  outline: none;
+  font-size: 20px;
+}
+
+.feed-comment-input-button {
+  width: 50px;
+  position: absolute;
+  right: 0px;
+  border: none;
+  background-color: white;
+  font-size: 17px;
+  opacity: 0.4;
+  cursor: pointer;
+  outline: none;
+}
+
+.main-right {
+  width: 300px;
+}
+
+.main-right-profiles {
+  display: flex;
+}
+
+.main-right-profiles img {
+  border-radius: 50%;
+}
+
+.main-right-profile-column {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  margin-left: 10px;
+}
+
+.main-right-profile-desc {
+  margin-top: 5px;
+}
+
+.main-right-stories {
+  margin-top: 20px;
+  border: 1px solid #dbdbdb;
+  border-radius: 5px;
+  background-color: white;
+}
+
+.main-right-story-header {
+  display: flex;
+  justify-content: space-between;
+  padding: 20px 20px;
```

아래와 같이 작성해주실 수 있겠네요! 
```suggestion
  padding: 20px;
```